module/Makefile.in

```diff
@@ -25,6 +25,24 @@ export ZFS_MODULE_CFLAGS ZFS_MODULE_CPPFLAGS
 
 SUBDIR_TARGETS = icp lua
 
+probe_files:
+	rm -f @top_srcdir@/module/zfs/dtrace_probes.c \
+		@abs_top_srcdir@/include/sys/dtrace_probe.h
```

Comment:
any reason for one of these to be singular `probe` and the other plural `probes`?

Reply:
No.  I'll go with plural since there are many probe functions.